src/main/java/com/homework/lesson12/Comment.java

```diff
@@ -0,0 +1,19 @@
+package com.homework.lesson12;
+
+import com.google.api.client.util.DateTime;
+import lombok.Data;
+import lombok.ToString;
+
+@ToString
```

Comment:
@Data is a convenient shortcut annotation that bundles the features of @ToString, @EqualsAndHashCode, @Getter / @Setter and @RequiredArgsConstructor together: In other words, @Data generates all the boilerplate that is normally associated with simple POJOs (Plain Old Java Objects) and beans: getters for all fields, setters for all non-final fields, and appropriate toString, equals and hashCode implementations that involve the fields of the class, and a constructor that initializes all final fields, as well as all non-final fields with no initializer that have been marked with @NonNull, in order to ensure the field is never null.
So, annotation @ToString is redundant.